ui/ecsconfig.py

```diff
@@ -480,25 +480,33 @@ def add_all():
         return None
 
     if l:
```

Comment:
o is my default (o)utput function.  It keeps formatting consistent across all utilities.  I'll also use v() to tag verbose outputs, d() for debug outputs, and so on.  We can talk about replacing that with something easier for other people to remember.